addons/website/static/src/js/editor/snippets.editor.js

```diff
@@ -170,5 +182,22 @@ weSnippetEditor.Class.include({
             tab: this.tabs.THEME,
         });
     },
+    /**
+     * Called when search input value changed > adapts the snippets grid.
+     */
+    _onSearchFilterInputChange: function () {
+        var self = this;
+        self._searchFilterValue = self.$('.o_snipet_search_filter_input').val().toLowerCase();
+        // show all snippets
+        self.$('.o_panel').show();
+        self.$('.oe_snippet').filter(function(){
```

Comment:
- Use a JS linter. There is a sample configuration here: https://github.com/odoo/odoo/wiki/Javascript-coding-guidelines
- Here, use the arrowed syntax to declare your anonymous function. This has the advantage to be able to use the "parent" `this` inside. So instead of 
```javascript
$els.filter(function () {
    var $snippet = $(this);
    self.hello();
```
Use
```javascript
$els.filter((i, el) => {
    const $snippet = $(el);
    this.hello();
```
-> See also the use of `const` when you can, `let` at worst (probably never `var`).